mappings/net/minecraft/block/DoorBlock.mapping

```diff
@@ -8,14 +8,14 @@ CLASS net/minecraft/class_2323 net/minecraft/block/DoorBlock
 	FIELD field_10944 EAST_SHAPE Lnet/minecraft/class_265;
 	FIELD field_10945 OPEN Lnet/minecraft/class_2746;
 	FIELD field_10946 HALF Lnet/minecraft/class_2754;
-	METHOD method_10032 getCloseSoundEventId ()I
+	METHOD method_10032 getOpenSoundEventId ()I
 	METHOD method_10033 setOpen (Lnet/minecraft/class_1297;Lnet/minecraft/class_1937;Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;Z)V
 		ARG 1 entity
 		ARG 2 world
 		ARG 3 state
 		ARG 4 pos
 		ARG 5 open
-	METHOD method_10034 getOpenSoundEventId ()I
+	METHOD method_10034 getCloseSoundEventId ()I
 	METHOD method_10035 getHinge (Lnet/minecraft/class_1750;)Lnet/minecraft/class_2750;
 		ARG 1 ctx
 	METHOD method_10036 playOpenCloseSound (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Z)V
```

Comment:
Ignore me if it's out of scope, but `playOpenCloseSound` is a grammatically weird name. I'd rename it to `playOpenOrCloseSound` or `playToggleSound`.

Reply:
I mean, it's not that odd if you think of it as "play (the) open/close sound"